timemachine/fe/rbfe.py

```diff
@@ -620,11 +638,10 @@ def run_pair(mol_a, mol_b, core, forcefield, protein, n_frames, seed, n_eq_steps
         core,
         forcefield,
         complex_host_config,
-        seed + 1,
+        seed,  # this was seed + 1, why?
```

Comment:
this was to avoid complex and solvent re-using the same seed. Not sure if it matters at all in practice.

Reply:
I doubt it would, since the dof are different anyways